### PR TITLE
add redirect for all other frontend request.

### PR DESCRIPTION
**Pull request by ngongoll**
i added a redirect for all frontend calls. 
in backend config you are able to exclude paths if you need to. default is checkout/onepage and all calls for Divante_VueStorefrontExternalCheckout and Divante_VueStorefrontBridge module

### CODE REVIEW

**Comment by ResuBaka** [comment]
So this is so you can redirect all calls that go magento to the vsf frontend home page that you didn't exclude in the config?

Am I understanding that right?

**Comment by ngongoll** [comment]
yes that's right

File: app/code/community/Divante/VueStorefrontExternalCheckout/etc/config.xml [hunk 1]
```diff
@@ -39,5 +51,13 @@
             </checkout_onepage_controller_success_action>
         </events>
     </global>
+    <default>
+        <vuestorefrontexternalcheckout>
+            <vuestorefrontexternalcheckout_group>
+                <exclude_route><![CDATA[checkout/onepage]]></exclude_route>
+                <externalcheckout_url>order-success</externalcheckout_url>
```
Comment (ResuBaka) [inline app/code/community/Divante/VueStorefrontExternalCheckout/etc/config.xml]: I would still prefer when this is empty as it other wise would be a breaking change for stores as it was previews an empty string so you were not redirected to vsf success page.

**Review by ResuBaka — CHANGES_REQUESTED** [review]

**Comment by ResuBaka** [comment]
I only found one thing, otherwise it is a great feature to have. 

I can only see that some people could break there payment provider or other modules when they use a frontend controller. But there I think they need to check it for them self it it breaks something or not.

**Review by ResuBaka — APPROVED** [review]

**Pull request merged**